src/app/epilepsy/pages/charts/chart-seizures-by-reason/chart-seizures-by-reason.component.html

```
@@ -0,0 +1,13 @@
+<div *ngIf="chartOptions$ | async as chartOptions; else errorTpl">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)